assets/cubyz/sbb/oak/1/trunk_6.zig.zon

```diff
@@ -0,0 +1,21 @@
+.{
+	.blueprint = "cubyz:oak/1/trunk_6",
+	.children = .{
+		.blue = .{
+			.{.structure = "cubyz:oak/1/branch_1", .chance = 0.333},
+			.{.structure = "cubyz:oak/1/branch_2", .chance = 0.333},
+			.{.structure = "cubyz:oak/1/branch_3", .chance = 0.334},
+		},
+		.crimson = .{
+			.{.structure = "cubyz:oak/1/tree_blob_1", .chance = 0.5},
+			.{.structure = "cubyz:oak/1/tree_blob_3", .chance = 0.5},
+		},
+		.yellow = .{
+			.{.structure = "cubyz:oak/1/stub_1", .chance = 0.1},
+			.{.structure = "cubyz:oak/1/stub_2", .chance = 0.1},
+			.{.structure = "cubyz:oak/1/stub_3", .chance = 0.1},
+			.{.structure = "cubyz:oak/1/stub_4", .chance = 0.1},
+			.{.structure = "cubyz:null/null_side", .chance = 0.6},
```

Comment:
@Argmaster could you please work on #1250 before we have too many of these?

Reply:
Ye I will have it done today.

Reply:
#1388

Reply:
merged, so please remove all the null files